docs/development/docs/frontend/testing.md

```diff
@@ -0,0 +1,92 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Frontend Testing
+
+## Integration testing
```

Comment:
Is "Unit testing" not the better term for it? It doesn't really test the integration (as it is not connected to the backend)

Reply:
It depends on the test you write. You can also use it to test the integration of different components and services. This article nicely describes the different test types that are possible (https://vsavkin.com/three-ways-to-test-angular-2-components-dcea8e90bd8d). But I agree that only naming it Integration test is confusing, so maybe change it to “Unit & Integration testing“?

Reply:
"Unit & Integration testing" sounds good.